src/elements/content-sidebar/activity-feed/annotations/__tests__/__snapshots__/AnnotationActivity.test.js.snap

```diff
@@ -0,0 +1,185 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`elements/content-sidebar/ActivityFeed/annotations/AnnotationActivity should correctly render annotation activity 1`] = `
```

Comment:
I think we should avoid large snapshots, as they tend to be overlooked. Can we define explicit assertions in our tests, instead?